File: decompiled/sources/com/hjq/toast/config/IToastStrategy.java
package com.hjq.toast.config;

import android.app.Application;

public interface IToastStrategy {
    void bindStyle(IToastStyle<?> iToastStyle);

    void cancelToast();

    IToast createToast(Application application);

    void registerStrategy(Application application);

    void showToast(CharSequence charSequence);
}

File: decompiled/sources/com/hjq/toast/config/IToastStrategy.kt
package com.hjq.toast.config

import android.app.Application

interface IToastStrategy {
    fun bindStyle(style: IToastStyle<*>)

    fun cancelToast()

    fun createToast(application: Application): IToast

    fun registerStrategy(application: Application)

    fun showToast(text: CharSequence)
}
